app/models/mediaflux/http/request.rb

```diff
@@ -142,6 +142,13 @@ def xtoshell_xml( name: self.class.service)
         xml.strip.gsub("\"","'").gsub("<args>","").gsub("</args>","")
       end
 
+      # This method is used for transforming iso8601 dates to dates that MediaFlux likes
```

Comment:
Don't we have a method that does this already? https://github.com/pulibrary/tiger-data-app/blob/main/app/models/mediaflux_time.rb#L24-L30